tauri/helpers/generator.js

```diff
@@ -0,0 +1,13 @@
+const compileTemplate = require('lodash.template'),
+  fs = require('fs'),
+  appPaths = require('./app-paths'),
+  path = require('path')
+
+module.exports = cfg => {
+  const apiTemplate = fs.readFileSync(path.resolve(__dirname, '../../lib/tauri.js'), 'utf-8')
```

Comment:
I don't like these relative paths...

Reply:
me neither but what can we do?

Reply:
Use apppaths / parse from its expected entry point in node_modules?

Reply:
But its not a blocker

Reply:
appPaths are for devland paths so won't work here.. I don't see how the options are better here